src/containers/EventsHandlers/market.ts

```diff
@@ -123,17 +123,54 @@ const MarketEvents = () => {
           if (typeof pool === 'undefined') {
             return
           }
-          marketProgram
-            .onTickmapChange(new PublicKey(address), tickmap => {
-              dispatch(
-                actions.updateTickmap({
-                  address: address,
-                  bitmap: tickmap.bitmap
-                })
-              )
-            })
-            .then(() => {})
-            .catch(() => {})
+          console.log(pool)
+          // trunk-ignore(eslint/@typescript-eslint/no-floating-promises)
+          marketProgram.onTickmapChange(new PublicKey(address), tickmap => {
+            const changes = findTickmapChanges(
+              tickmaps[address].bitmap,
+              tickmap.bitmap,
+              pool.currentTickIndex
+            )
+            for (const [index, info] of Object.entries(changes)) {
+              if (info === 'added') {
+                // trunk-ignore(eslint/@typescript-eslint/no-floating-promises)
+                marketProgram.onTickChange(
+                  new Pair(pool.tokenX, pool.tokenY, { fee: pool.fee.v }),
+                  +index,
+                  tickObject => {
+                    dispatch(
+                      actions.updateTicks({
+                        address: pool.address.toString(),
+                        index: +index,
+                        tick: tickObject
+                      })
+                    )
+                  }
+                )
+              } else {
+                // trunk-ignore(eslint/@typescript-eslint/no-floating-promises)
+                marketProgram.unsubscribeTick(
+                  new Pair(pool.tokenX, pool.tokenY, { fee: pool.fee.v }),
+                  +index,
+                  tickObject => {
```

Comment:
There is no fn: callback in `market.unsubscribeTick` function. I accidently added this parameter to `unsubscribeTick`. Solana web3 doesn't provide callback after unsubscribe. This function will be looks that in `0.5.30` SDK version:
`public async unsubscribeTick(pair: Pair, index: number)`.

Remove explicitly this tick from redux after `unsubscribeTick`